ykrt/pass_finder/run_tests.sh

```diff
@@ -1,7 +1,24 @@
 #!/bin/sh
+# This script runs tests cases for YK and YKLUA. The aim is
+# to feed this test script to try_passes.py and get list of
+# passes which are successful for all test suites. We can expand 
+# this script to accomodate new test suites in future.
+
+# how do I pass mode for yk through terminal?
```

Comment:
Not sure I understand the question. Maybe @vext01 does?

Reply:
I don't understand either :)

Please can you rephrase the question?

Reply:
@vext01 with current script, we just enter YK repository and run `cargo test`. I was wondering, while testing should the default mode be `Release` (because your original test script for YKLUA sets the mode as `Release`)? If yes, then I don't know how to set the mode to `Release` for YK.

Reply:
yes, thank you!